components/management/commands/load_components.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+from pathlib import Path
+
+from pydantic import ValidationError
+from django.core.management.base import BaseCommand, CommandError
+
+from blueprintapi.oscal.component import Model as ComponentModel
+from catalogs.models import Catalog
+from components.models import Component
+
+
+class Command(BaseCommand):
+    help = (
+        "Ingest component data into the Component table. "
+        "Currently assumes a single component definition defined against a single catalog"
+    )
+
+    def add_arguments(self, parser):
+        parser.add_argument("--load-test-data", type=bool, default=True)
+        parser.add_argument("--catalog", type=str, default="CMS_ARS_3_1_MODERATE")
+
+    def handle(self, *args, **options):
+        if options["load_test_data"]:
+            self._load_test_data(**options)
```

Comment:
I know this might look odd/thin, but the idea was that we'd might want to extend this to load components from another source or something. Even directly from the component repo.